src/main/java/org/dependencytrack/notification/publisher/JiraPublisher.java

```diff
@@ -0,0 +1,179 @@
+        logger.info("Preparing to publish JIRA notification");
+        if (config == null) {
+            logger.warn("No configuration found. Skipping notification.");
+            return;
+        }
+        
+        try (QueryManager qm = new QueryManager()) {
+            jiraUsername = qm.getConfigProperty(JIRA_USERNAME.getGroupName(), JIRA_USERNAME.getPropertyName()).getPropertyValue();
+            final String password = qm.getConfigProperty(JIRA_PASSWORD.getGroupName(), JIRA_PASSWORD.getPropertyName()).getPropertyValue();
+            jiraPassword = (password != null) ? DataEncryption.decryptAsString(password) : null;
+            final String configDestination = config.getString("destination");
+            URL url;
+            String destination;
+            url = new URL(configDestination);
+            destination = url.getProtocol()+"://"+url.getAuthority()+"/rest/api/2/issue";
+            final String[] pathElements = url.getPath().split("/");
+            if(pathElements.length == 0) {
+                logger.error("No Jira Project given");
+                return;
+            }
+            jiraProjectKey = pathElements[pathElements.length-1];
+            final String content = prepareTemplate(notification, template);
+            if (destination == null || content == null) {
+                logger.warn("A destination or template was not found. Skipping notification");
+                return;
+            }
+            logger.info("Template: "+content);
+            final String userAgent = "ui-dependency-track";
+            URL jiraUrl;
+            jiraUrl = new URL(destination);
+            HttpsURLConnection con;
+            con = (HttpsURLConnection) jiraUrl.openConnection();
+            con.setRequestMethod("POST");
+            con.setRequestProperty("User-Agent", userAgent);
+            con.setRequestProperty("Authorization", "Basic " + Base64.getEncoder().encodeToString((jiraUsername+":"+jiraPassword).getBytes()));
+            con.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
+            con.setDoOutput(true);
+            DataOutputStream wr = new DataOutputStream(con.getOutputStream());
+            wr.write(content.getBytes(StandardCharsets.UTF_8));
+            wr.flush();
+            wr.close();
+            int responseCode = con.getResponseCode();
+            logger.info("jira issue creation response code: "+responseCode);
+            if(responseCode == HttpURLConnection.HTTP_CREATED) {
+                BufferedReader in = new BufferedReader(
+                        new InputStreamReader(con.getInputStream()));
+                String inputLine;
+                StringBuffer response = new StringBuffer();
+    
+                while ((inputLine = in.readLine()) != null) {
+                    response.append(inputLine);
+                }
+                in.close();
+                logger.info("jira issue creation response body: "+response.toString());
```

Comment:
Please remove or change log level to `debug`.

Reply:
graded down to "debug"